Source/ARTInternalLog.m

```diff
@@ -1,36 +1,46 @@
 #import "ARTInternalLog.h"
+#import "ARTInternalLogBackend.h"
 #import "ARTVersion2Log.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 @interface ARTInternalLog ()
 
-@property (nonatomic, readonly) id<ARTVersion2Log> logger;
+@property (nonatomic, readonly) id<ARTInternalLogBackend> backend;
```

Comment:
The word "Backend" is mostly associated with a server side code, I wonder why it was chosen?

Reply:
Fair point — I didn't have a good word and had seen "backend" being used a lot in https://github.com/apple/swift-log, so thought I’d steal that. But I’ve changed it to `ARTInternalLogCore`, what do you think?

Reply:
`ARTInternalLogCore` is much better, thanks 👍